OlxProject/reposes/search_repos.rb

```diff
@@ -12,7 +12,7 @@ def initialize (searchItem, city)
     @city = city
   end
 end
-$searhSuccessfulItem = SearchItem.new("MacBook")
-$searhUnsuccessfulItem = SearchItem.new("qweqwe")
+$searh_successfu_item = SearchItem.new("MacBook")
+$searh_unsuccessful_item = SearchItem.new("qweqwe")
 
 $top_search_item = TopSearchItem.new("MacBook","Львовская область")
```

Comment:
why is everything global?

Reply:
I agree. I`ll make all them as local variables.